src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import * as S from './SubscriberGdpr.styles';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { InjectAppServices } from '../../../services/pure-di';
+
+const PermissionValue = ({ value }) => {
+  const permissionValue = value.toLowerCase();
+  const iconColor =
+    permissionValue === 'none' ? 'grey' : permissionValue === 'true' ? 'green' : 'red';
+  const valueText =
+    permissionValue === 'none'
+      ? 'Sin respuesta'
```

Comment:
This texts should be in the language file, as they were in the older component, like [this](https://github.com/FromDoppler/doppler-webapp/blob/7abe099ac32352ee7fd35c05b275ff74ad8a3c26/src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js#L95)  The key you have for this is `subscriber_gdpr.value_none` and you can have it programatically applied also using intl hook. An example of how we use it [here](https://github.com/FromDoppler/doppler-webapp/blob/7abe099ac32352ee7fd35c05b275ff74ad8a3c26/src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js#L16) and then in js [here](https://github.com/FromDoppler/doppler-webapp/blob/7abe099ac32352ee7fd35c05b275ff74ad8a3c26/src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js#L60)

Reply:
Done. Resolving conversation...